mixins/database.mixin.ts

```diff
@@ -1,18 +1,22 @@
-"use strict";
+'use strict';
 
-import _ from "lodash";
-import { Context } from "moleculer";
-import filtersMixin from "moleculer-knex-filters";
-import config from "../knexfile";
-const DbService = require("@moleculer/database").Service;
+import _ from 'lodash';
+import { Context } from 'moleculer';
+import filtersMixin from 'moleculer-knex-filters';
+import config from '../knexfile';
+const DbService = require('@moleculer/database').Service;
+
+export const MaterializedView = {
+  OBJECTS: 'objects',
```

Comment:
cia `publishing.objects` turetu but. paziurek i `tiles.objects.service.ts` koks collection'as + tenais galima butu naudot `MaterializedView.OBJECTS` vietoj collection taip pat. del to ir nesinaujina views'as refresh'o metu (meta error DB kad neegzistuoja toks daiktas kaip `public.objects`)